metasrc/tex/lsstdoc.py

```diff
@@ -185,6 +190,41 @@ def is_draft(self):
             return False
 
     @property
+    def revision_datetime(self):
+        """Current revision date of the document (`datetime.datetime`).
```

Comment:
Do we need a note here to point out that if you happen to have a python process that runs as people are editing the git repo the information will be out of date. I know that this can't happen if Travis is triggering this job but I wonder if we need to explicitly state somewhere that this all presupposes a static git repo.